x/records/keeper/keeper.go

```diff
@@ -35,6 +35,7 @@ type (
 		TransferKeeper     ibctransferkeeper.Keeper
 		IBCKeeper          ibckeeper.Keeper
 		ICACallbacksKeeper icacallbackskeeper.Keeper
+		IBCScopperKeeper   capabilitykeeper.ScopedKeeper
```

Comment:
nit type: `IBCScopperKeeper` → `IBCScopedKeeper`, same with rest of file
```suggestion
		IBCScopperKeeper   capabilitykeeper.ScopedKeeper
```

Reply:
done